src/js/notfound.js

```diff
@@ -0,0 +1,42 @@
+import { h } from 'preact'
+
+var quotearray = ["God help us; we&#39;re in the hands of engineers.",
```

Comment:
IMHO according to me it looks more consistent when
```js
var quotearray = [
  "msg1",
  "msg2"
]
```

The idea is to have single line objects in one line and multiple line objects in individual lines. It just gives a feel of consistency